Lib/subprocess.py

```diff
@@ -2061,7 +2061,13 @@ def send_signal(self, sig):
             # The race condition can still happen if the race condition
             # described above happens between the returncode test
             # and the kill() call.
-            os.kill(self.pid, sig)
+            try:
+                os.kill(self.pid, sig)
+            except ProcessLookupError as e:
+                # supress the process not found error in case the race
+                # condition happens, see bpo-40550
+                if e.errno != errno.ESRCH:
```

Comment:
This is harmless but redundant.  ProcessLookupError only occurs when `e.errno == ESRCH`.

https://github.com/python/cpython/blob/v3.10.0a2/Objects/exceptions.c#L2653

(i'll submit an edit turning this into a mere `pass`)

Reply:
Ah, gotcha!